src/index.html

```diff
@@ -9,10 +9,51 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="./styles/style.css"
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
Please check how class name for block with modifier should look like according to BEM. Here, it's not used as expected for all the `stars` blocks.

Reply:
UPD. Please ignore this comment. I was referring to the guideline different from the one used for the studies. I'm sorry for the confusion.

Reply:
@JohnyJDX 
According to the guideline used for the studies:
> Follow naming convention block__element--modifier

```
<div class="block-name block-name--modifier-name--modifier-value">
  <p class="block-name__element-name block-name__element-name--modifier-name">
    text
  </p>
</div>
```
So the syntax you used is correct. Please ignore the first comment of this thread. Sorry for confusion.